protocols/gossipsub/src/peer_score.rs

```diff
@@ -594,7 +595,12 @@ impl PeerScore {
 
         // this should be the first delivery trace
         if record.status != DeliveryStatus::Unknown {
-            tracing::warn!("Unexpected delivery trace: Message from {} was first seen {}s ago and has a delivery status {:?}", from, record.first_seen.elapsed().as_secs(), record.status);
+            tracing::warn!(
+                peer=%from,
+                status=?record.status,
+                "Unexpected delivery trace: Message from peer was first seen {}s ago", 
+                record.first_seen.elapsed().as_secs(),
```

Comment:
`Duration`s are actually best logged via `Debug`! That implements a human-friendly formatter that computes the correct precision based on the value.

Let's make that a field too please!